### DISCUSSION TITLE
Typos

### DISCUSSION THREAD

**Discussion by Zayd-R** [spamegg1/Epp-Discrete-Math-5th-solutions#6] [General]
Exercise SET 5.7
 
exercise 9 : guess should be  gn =1/( (2^n) - 1)

exercise  28 - 32:  the number in the statement (k > base) base not there

exercise  35: the last equation at the end of the proof should be 2^(k + 2)  - (-1)^(k +2) / 3

**Comment by spamegg1** [comment]
Thanks! Fixed 9 and 35.
> exercise 28 - 32: the number in the statement (k > base) base not there

Not sure about this?

**Reply by Zayd-R** [reply]
in the statements  "Show that for any integer k ≥, if P(k) is true then P(k + 1) is true" you forgot to add the number in this part "k ≥ "

**Reply by spamegg1** [reply]
Gotcha. Thanks again! :heart:
